src/screens/ProfileScreen/index.js

```diff
@@ -106,7 +107,7 @@
     const currentTime = new Date().getTime();
     const id = feeds && feeds[viewPostTimeIndex]?.id;
     if (id) viewTimePost(id, currentTime - timer.getTime(), SOURCE_FEED_TAB);
-    navigator.navigate(NavigationConstants.CREATE_POST_SCREEN);
+    navigator.navigate(NavigationConstants.CREATE_POST_SCREEN, {});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The navigation function now includes an empty object as a second parameter. If no specific parameters are needed for the 'CREATE_POST_SCREEN', this change is unnecessary.
```diff
-    navigator.navigate(NavigationConstants.CREATE_POST_SCREEN);
+    navigator.navigate(NavigationConstants.CREATE_POST_SCREEN, {});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->